### ISSUE TITLE
`looponfail` continues to run tests after everything passes

### ISSUE THREAD

**Issue by OddBloke** [alfredodeza/pytest.vim#63]
Observed on the latest HEAD.

**Comment by OddBloke** [comment]
I can reproduce this with:

```
class TestThing(object):

    def test_thing(self):
        assert False
```

and then toggling False to True.

**Comment by alfredodeza** [comment]
Should be fixed with 1201491d4978ffef09126eacee011393b83ad284 

Can you check and confirm ? Thanks!

**Comment by OddBloke** [comment]
LGTM, thanks again for the speedy response!
